src/UI/Factory.php

```diff
@@ -336,4 +335,16 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     View Controls switch between different visualisation of data.
```

Comment:
Sounds good for me, but I would suggest start with type name.
"**Section** View Controls allow...."